ansible/roles/ocp-workload-istio-prometheus-demo/templates/istio_cleanup.sh

```diff
@@ -0,0 +1,7 @@
+for i in `oc get clusterroles | grep istio`; do
```

Comment:
You are not using `istio_cleanup.sh` anywhere. Can it be removed ?

Reply:
@cricci82 did you see my comment ?
Are you done with the PR ? I can remove the istio_cleanup.sh myself if you want.

Reply:
Hi sorry for the delayed response.  Yes I saw and as a result I’m making some additional cleanups but haven’t had an opportunity to submit the update yet.

Thanks for the input